extensions/shared/plan-upgrade-notification.js

```diff
@@ -15,9 +16,40 @@ if ( undefined !== typeof window && window.location ) {
 	const { query } = parseUrl( window.location.href, true );
 
 	if ( query.plan_upgraded ) {
-		dispatch( 'core/notices' ).createNotice(
-			'success',
-			__( 'Plan upgraded. You may use premium blocks now.', 'jetpack' )
-		);
+		const planUrl = `https://wordpress.com/plans/my-plan/${ window.location.host }`;
+
+		apiFetch( { path: '/jetpack/v4/site', parse: true } )
```

Comment:
Nit; since `parse` is true by default so would be ok to remove in here? :-)

https://developer.wordpress.org/block-editor/packages/packages-api-fetch/#parse-boolean-default-true